src/renderer/kube-object.store.ts

```diff
@@ -131,6 +132,12 @@ export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemSt
     }
   }
 
+  async loadSelectedNamespaces(): Promise<void> {
```

Comment:
The name seems a bit misleading, it's really `loadFromSelectedNamespaces` or `loadForSelectedNamespaces`. But those names are getting too long. Maybe not worth changing...

Reply:
how about `loadContextNamespaces()`?